packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -690,6 +690,9 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 	// Set the initial content and load the plugin CSS and JS files
 	// -----------------------------------------------------------------------------------------
 
+	const documentCssElements: Record<string, HTMLLinkElement> = {};
+	const documentScriptElements: Record<string, HTMLScriptElement> = {};
+
 	const loadDocumentAssets = (editor: any, pluginAssets: any[]) => {
 		// Note: The way files are cached is not correct because it assumes there's only one version
```

Comment:
Since you fixed this bug, please also update (or remove if no longer needed) this comment